sdk/identity/identity/README.md

```diff
@@ -122,6 +122,9 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+## Developer Contributing Guide
```

Comment:
Same comment as in the other PR regarding the header

Reply:
done